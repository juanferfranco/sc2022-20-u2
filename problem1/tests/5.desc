Adiciona 3 eventos y borra el del medio
