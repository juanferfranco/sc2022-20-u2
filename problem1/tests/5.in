add
ev1
add
ev2
add
ev3
rm
ev2
ask
exit
