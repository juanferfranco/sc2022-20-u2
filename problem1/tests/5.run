./main < tests/5.in
