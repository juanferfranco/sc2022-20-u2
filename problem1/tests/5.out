ev1
ev3
